homeassistant/components/prosegur/alarm_control_panel.py

```diff
@@ -69,7 +69,7 @@ def __init__(self, contract: str, auth: Auth) -> None:
             manufacturer="Prosegur",
             model="smart",
             identifiers={(DOMAIN, contract)},
-            configuration_url="https://smart.prosegur.com",
+            configuration_url="https://https://api-smart.prosegur.cloud",
```

Comment:
> Prosegur updated their API domain, which caused the integration to stop working. The old domain (smart.prosegur.com) has been replaced with api-smart.prosegur.cloud. This PR updates the configuration URL in the alarm_control_panel.py file to reflect this change, restoring functionality.

Nothing in this PR changes any functionality. All you updated was a link, and above all, you broke it.

Could you clarify what you are trying to do?

../Frenck